### PR TITLE
Fix ci attempt 2

### PR DESCRIPTION
**Pull request by brian6932**
Please run tests, before merging, so I don't have to open any more PRs 👍 
(not needed anymore, since no longer a first time contributor)

### CODE REVIEW

**Comment by brian6932** [comment]
well since ci is being changed, I'm not sure how useful previous checks are in this case 🤔

**Comment by brian6932** [comment]
ohhh, it's checking broken, ok, I'll try to fix that

**Comment by brian6932** [comment]
@AntonOks Ok it's fixed, make sure to move these dirs outside of the folder before pulling:
```gitignore
local/
ongoing/
broken/
```
You can paste them back in once done, it's just so that they're ignored properly, although broken isn't really needed, I just moved the single broken into deprecated.
Sry for not making it clear before that tests should've been ran 👍

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,37 +6,37 @@ on:
...
-#        run: |
-#          $env:SCOOP_HOME="$(Resolve-Path '.\scoop_core')"
-#          .\scoop_core\test\bin\init.ps1
-#          .\my_bucket\bin\test.ps1
+  test_powershell:
+    name: WindowsPowerShell
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Bucket
+        uses: actions/checkout@v2
```
Comment (AntonOks) [inline .github/workflows/ci.yml:14]: why back to v2?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -6,37 +6,37 @@ on:
...
-#          repository: ScoopInstaller/Scoop
-#          path: 'scoop_core'
-#      - name: Init and Test
-#        shell: powershell
-#        run: |
-#          $env:SCOOP_HOME="$(Resolve-Path '.\scoop_core')"
-#          .\scoop_core\test\bin\init.ps1
-#          .\my_bucket\bin\test.ps1
+  test_powershell:
+    name: WindowsPowerShell
```
Comment (AntonOks) [inline .github/workflows/ci.yml:10]: hmmm, I don't use nor recommend PowerShell. PWSH rules ;)

File: bin/Update-AutoupdateManifests.ps1 [hunk 3]
```diff
@@ -1,75 +0,0 @@
-<#
```
Comment (AntonOks) [inline bin/Update-AutoupdateManifests.ps1:1]: sorry, I'm aware this scripts doesn't fall under or fits to the SCOOP format and test but I still want it in my repo and under GIT...

File: bin/auto-pr.ps1 [hunk 4]
```diff
@@ -1,9 +1,9 @@
 param(
     # overwrite upstream param
-    [String]$upstream = "<username>/<bucketname>:main"
+    [String]$upstream = "AntonOks/scoop-aoks:master"
```
Comment (AntonOks) [inline bin/auto-pr.ps1:3]: thanks. fixed in a own commit now

File: deprecated/raidrive-aoks.json [hunk 5]
```diff
@@ -17,10 +17,10 @@
         ]
     ],
     "checkver": {
-        "url": "https://www.raidrive.com/download"
+        "url": "https://www.raidrive.com/download",
```
Comment (AntonOks) [inline deprecated/raidrive-aoks.json:20]: manifest is broken ;) anyhow, but thanks, fixed this now... just in case, you never know :)

File: .gitignore [hunk 6]
```diff
@@ -3,3 +3,6 @@
 ._*
 page.html
 checkver-*.html
+local/
```
Comment (AntonOks) [inline .gitignore:6]: sorry, will not make this change as I want to keep those dir's close to my scoop buckets and under GIT

Comment (brian6932) [reply .gitignore:6]: gitignore just doesn't commit them, it's not like they're not there 🤷 they just don't get flagged as broken

File: deprecated/raidrive-aoks.json (hunk 5 above)
Comment (brian6932) [reply deprecated/raidrive-aoks.json:20]: I'm sry what is the problem here? just a comma fix, I didn't even make it, it was auto formatted
It was flagged as breaking actions

File: .github/workflows/ci.yml (hunk 1 above)
Comment (brian6932) [reply .github/workflows/ci.yml:14]: I just mirrored ci template from upstream, you can suggest a change in the PR you know, also it was originally commented out :\

File: .github/workflows/ci.yml (hunk 2 above)
Comment (brian6932) [reply .github/workflows/ci.yml:10]: It's good to have it for backwards compatibility I guess, doesn't really matter to me, I can remove it, again mirrored ci template from upstream

File: bin/Update-AutoupdateManifests.ps1 (hunk 3 above)
Comment (brian6932) [reply bin/Update-AutoupdateManifests.ps1:1]: Not sure what the issue is here either :\ It's an auto formatter, you're supposed to always have an extra line at the end of anything committed to git, this is just a unix norm

**Comment by AntonOks** [comment]
@brian6932 Thanks for all your findings and fixes... before I didn't really care about the "ci.yml" aka "Test" workflow and only wanted to have the "Excavator" aka "autoupdate" feature... anyhow, due to your feedback I've now switched to an [own Scoop clone](https://github.com/AntonOks/scoop-aoks/commit/791ea269cf2fbedaed2f52b2471d741f0fda4135) and adopted the "File-Import" to exclude some not-json aka non-manifests files [here](https://github.com/AntonOks/Scoop/commit/6b94a4730e8389ce2ad1a9a428f4e52b3b49938c) and [here](https://github.com/AntonOks/Scoop/commit/0943ec8bd95a52c184d42de766137d86bb956804).

Long story short, the "Tests" workflow runs fine now as well and should only check json/manifest files, see [here](https://github.com/AntonOks/scoop-aoks/actions/runs/2341389160)
